server/test/integration/dashboardTest/submitEditClientTest.js

```diff
@@ -134,6 +138,7 @@ describe('dashboard.js integration tests: submitEditClient', () => {
       this.isDisplayed = '    true    '
       this.isSendingAlerts = '    true    '
       this.isSendingVitals = '    true    '
+      this.language = 'en'
```

Comment:
This test is making sure that all values are trimmed before they go into the database. Do you see how all the other `this.<something>` values have extra whitespace around their values? Could you please do that here too to make sure that it will also test that `this.language` is being trimmed as expected?